### ISSUE TITLE
Bugs blocking RF-DETR TFlite conversion

### ISSUE THREAD

**Issue by EHxuban11** [PINTO0309/onnx2tf#936]
### Issue Type

Others

### OS

Windows

### onnx2tf version number

2.4.2

### onnx version number

1.20.1

### onnxruntime version number

1.24.3

### onnxsim (onnx_simplifier) version number

0.6.5

### tensorflow version number

2.21.0

### Download URL for ONNX

https://huggingface.co/LibreYOLO/LibreRFDETRn/resolve/main/LibreRFDETRn.onnx

### Parameter Replacement JSON

```json
{
    "format_version": 1,
    "operations": [
      {
        "op_name":
  "wa/model/backbone/backbone.0/encoder/encoder/embeddings/Add",
        "param_target": "inputs",
        "param_name":
  "wa/model/backbone/backbone.0/encoder/encoder/embeddings/Concat_1_output_0",
        "pre_process_transpose_perm": [0, 1, 2]
      },
      {
        "op_name":
  "wa/model/backbone/backbone.0/encoder/encoder/embeddings/Add",
        "param_target": "inputs",
        "param_name":
  "model.backbone.0.encoder.encoder.embeddings.position_embeddings",
        "pre_process_transpose_perm": [0, 2, 1]
      }
    ]
  }
```

### Description

Hello, thank you for your work on this library. 

I'm adding TFLite export to LibreYOLO  (https://github.com/LibreYOLO/libreyolo) for RF-DETR models. 

There are three separate bugs preventing it:
1) Tile.py line 122. When the multiples input is a numpy array, the code does range(len(input_tensor_2.shape)) instead of range(len(input_tensor_2)). For a 1-D multiples array like [4, 1, 1], .shape is (3,) so len(.shape) is 1 instead of 3. This produces a broken initial permutation and the conversion produces wrong outputs.

2) Slice.py, two separate lines. tf.clip_by_value(t=begin_, clip_value_min=0, clip_value_max=1) and the same for end_ crash because begin_ and end_ are int64 but the integer scalar bounds resolve to int32. Fix is to cast: tf.cast(begin_, tf.int32) and tf.cast(end_, tf.int32).

3) LayerNormalization.py. RF-DETR's DINOv2 backbone uses windowed attention which produces a normalisation axis that is not statically known at graph build time. tf_keras.layers.LayerNormalization(axis=axis)(input_tensor) throws TypeError or ValueError when axis is dynamic. Fix is a try/except that falls back to tf.nn.moments for those cases.

I forked the library to https://github.com/EHxuban11/onnx2tf and fixed this issue. I will now open a PR in case you want to change it or take a look at the diff.

**Comment by EHxuban11** [comment]
PR: https://github.com/PINTO0309/onnx2tf/pull/937

**Comment by PINTO0309** [comment]
Accessing the link below results in an "Entry not found" error. Is there an accessible ONNX file?

https://huggingface.co/LibreYOLO/LibreRFDETRn/resolve/main/LibreRFDETRn.onnx

<img width="683" height="136" alt="Image" src="https://github.com/user-attachments/assets/7605f715-e5dc-4d6d-8181-6926922af044" />

**Comment by EHxuban11** [comment]
Hello @PINTO0309 , my bad. You can check again: https://huggingface.co/LibreYOLO/LibreRFDETRn/resolve/main/LibreRFDETRn.onnx

**Comment by PINTO0309** [comment]
Thanks.

https://github.com/PINTO0309/onnx2tf/releases/tag/2.4.3
